src/mercury-inventory/mercury/inventory/hooks.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2015 Jared Rodriguez (jared.rodriguez@rackspace.com)
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import logging
```

Comment:
I know this is ironic coming from me, but unit tests should be added to cover hooks.py